src/services/repo/repo.service.ts

```diff
@@ -142,4 +135,11 @@ export class RepoService {
       }),
     );
   }
+
+  private getRepoMainPath(remote: string, namespace?: string) {
```

Comment:
what is the purpose of namespace

Reply:
single repos vs compare repos, just for group